### PR TITLE
Update replace element UI VC-1392

### PR DESCRIPTION
**Pull request by konutis**

### CODE REVIEW

**Review by pavelthq — APPROVED** [review]

File: public/sources/less/attributes/element/styles.less [hunk 1]
```diff
@@ -33,20 +47,21 @@
...
+  flex-wrap: wrap;
+  display: flex;
+  padding: 0;
+  justify-content: flex-start;
 
-  .vcv-ui-item-list-item {
-    display: inline-block;
-    margin: 0 7px;
-    vertical-align: top;
+  //list-style-type: none;
```
Comment (nikitahl) [inline public/sources/less/attributes/element/styles.less:57]: Are these comments necessary?

File: public/sources/less/attributes/element/styles.less [hunk 2]
```diff
@@ -33,20 +47,21 @@
...
+  //padding: 40px 2px 0;
+  //margin: 0;
 
-    &:first-child {
-      margin-left: 0;
-    }
+  .vcv-ui-item-list-item {
+    //display: inline-block;
```
Comment (nikitahl) [inline public/sources/less/attributes/element/styles.less:62]: Are these comments necessary?

File: public/sources/attributes/element/Component.js [hunk 3]
```diff
@@ -139,126 +147,148 @@ export default class ElementAttribute extends Attribute {
...
+            label: cookElement.get('name'),
+            value: tag
+          }
+        }
+      )
+      replacementBlock = (
+        <div className='vcv-ui-form-group vcv-ui-replace-element-block-dropdown'>
+          <Dropdown
+            fieldKey='replaceElement'
+            updater={this.onClickReplacementDropdown.bind(this)} // TODO: Tag
```
Comment (nikitahl) [inline public/sources/attributes/element/Component.js:262]: What is the point of this TODO? What needs to be done? Can it be done within this task?

File: public/sources/attributes/element/Component.js [hunk 4]
```diff
@@ -139,126 +147,148 @@ export default class ElementAttribute extends Attribute {
...
-    const elementLabel = (tabLabel && tabLabel.toLowerCase()) || category.toLowerCase() || 'element'
     const categorySettings = hubCategoriesService.get(category)
-    const localizations = window.VCV_I18N && window.VCV_I18N()
-    const replaceElementText = localizations.replaceElementText.split('{elementLabel}').join(elementLabel)
     let replacementBlock = ''
 
-    if (!disableReplaceable) {
-      if (categorySettings && this.state.showReplacements) {
-        const replacementItemsOutput = categorySettings.elements.map((tag) => {
+    // TODO: show another wrapper
```
Comment (nikitahl) [inline public/sources/attributes/element/Component.js:244]: Is this TODO done or needs to be done? Can it be done within this task?

File: public/sources/attributes/element/Component.js [hunk 5]
```diff
@@ -139,126 +147,148 @@ export default class ElementAttribute extends Attribute {
...
+  getReplaceShownStatus (category) {
+    const categorySettings = hubCategoriesService.get(category)
+    let showElementReplaceIcon = false
+    const presetsByCategory = hubElementsStorage.action('getPresetsByCategory', category)
+
+    if (presetsByCategory.length) {
+      showElementReplaceIcon = true
+    }
+
+    if (!showElementReplaceIcon && categorySettings && categorySettings.elements && categorySettings.elements.length > 1) {
```
Comment (nikitahl) [inline public/sources/attributes/element/Component.js:224]: What if there's 1 element? Then replace won't be shown?

File: public/components/panels/editForm/lib/editForm.js [hunk 6]
```diff
@@ -174,22 +182,68 @@ export default class EditForm extends React.Component {
...
+  getReplaceShownStatus (category) {
+    const categorySettings = hubCategoriesService.get(category)
+    let showElementReplaceIcon = false
+    const presetsByCategory = hubElementsStorage.action('getPresetsByCategory', category)
+
+    if (presetsByCategory.length) {
+      showElementReplaceIcon = true
+    }
+
+    if (!showElementReplaceIcon && categorySettings && categorySettings.elements && categorySettings.elements.length > 1) {
```
Comment (nikitahl) [inline public/components/panels/editForm/lib/editForm.js:210]: What if there's 1 element? Then replace won't be shown? Or is it the same one element?

Comment (konutis) [reply public/components/panels/editForm/lib/editForm.js:210]: It is the same one element.

File: public/sources/attributes/element/Component.js (hunk 5 above)
Comment (konutis) [reply public/sources/attributes/element/Component.js:224]: It is the same one element.

File: public/sources/less/attributes/element/styles.less (hunk 1 above)
Comment (konutis) [reply public/sources/less/attributes/element/styles.less:57]: No, will remove

File: public/sources/less/attributes/element/styles.less (hunk 2 above)
Comment (konutis) [reply public/sources/less/attributes/element/styles.less:62]: No, will remove

File: public/sources/attributes/element/Component.js (hunk 3 above)
Comment (konutis) [reply public/sources/attributes/element/Component.js:262]: This was already there. I can remove TODO

File: public/sources/attributes/element/Component.js (hunk 4 above)
Comment (konutis) [reply public/sources/attributes/element/Component.js:244]: This was already there. I can remove TODO

**Review by nikitahl — APPROVED** [review]

**Pull request merged**